lib/DonorsList/DonorsContainer.js

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'react-final-form-arrays';
+
+import {
+  Col,
+  Loading,
+  Row,
+} from '@folio/stripes/components';
+
+import DonorsList from './DonorsList';
+import { useFetchDonors } from './hooks';
+
+function DonorsContainer({ name, donorOrganizationIds }) {
+  const [donorIds, setDonorIds] = useState(() => donorOrganizationIds);
+  const { donors, isLoading } = useFetchDonors(donorIds);
+
+  const handleFetchDonors = useCallback(ids => {
+    setDonorIds(ids);
+  }, []);
+
+  const donorsMap = donors.reduce((acc, contact) => {
+    acc[contact.id] = contact;
+
+    return acc;
+  }, {});
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  return (
+    <Row>
+      <Col xs={12}>
+        <FieldArray
+          name={name}
+          id={name}
+          component={DonorsList}
+          fetchDonors={handleFetchDonors}
```

Comment:
should be called like `setDonorIds` and accept `setDonorIds`, because now the callback does not fetch anything

Reply:
updated